test/parallel/test-runner-output.mjs

```diff
@@ -171,7 +171,12 @@ const tests = [
     name: 'test-runner/output/source_mapped_locations.mjs',
     flags: ['--test-reporter=tap'],
   },
-  process.features.inspector ? { name: 'test-runner/output/lcov_reporter.js', transform: lcovTransform } : false,
+  process.features.inspector ?
```

Comment:
Hey @cjihrig, not yet.  
Currently, since our tests are located under the `test/**` path, they are excluded by default from the coverage output.  

I was planning to add a test to validate that defaults are being set, but it seems I'll need to test the `parseCommandLine()` utility instead. Do you think that’s enough?  

I don’t think we currently have a way to assert test configurations (I’m not sure about this, I’ll take a look tonight).

Reply:
> Currently, since our tests are located under the test/** path, they are excluded by default from the coverage output.

What if you use the [`tmpdir`](https://github.com/nodejs/node/tree/main/test/common#tmpdir-module) module, copy files into that, and use that as the working directory?

Reply:
sure, makes sense 🚀 !
I'll take a look asap!

Reply:
done!